crates/ruff_linter/src/rules/pyupgrade/fixes.rs

```diff
@@ -25,7 +25,7 @@ pub(crate) fn remove_import_members(
 
     // Reconstruct the source code by skipping any names that are in `members`.
     let mut output = String::with_capacity(import_from_stmt.range().len().to_usize());
-    let mut last_pos = TextSize::default();
+    let mut last_pos = import_from_stmt.start();
```

Comment:
Now, we use the locator containing the entire source code, so the start offset should be the statement start offset.